astropy/units/quantity.py

```diff
@@ -168,7 +168,7 @@ def new_like(self, cols, length, metadata_conflicts='warn', name=None):
 
         Returns
         -------
-        col : Quantity (or subclass)
+        col : `Quantity` (or subclass)
```

Comment:
Are you sure the bare `Quantity` work? I've had problems before when it is imported/subclasses elsewhere. Maybe just `~astropy.units.Quantity` for consistency throughout?

Reply:
I agree. I've had this problem with SkyCoord. [reminder @nstarman push a fix for subclassing SkyCoord when x-ref turned on].

Reply:
✅